plugins/modules/os10_command.py

```diff
@@ -17,7 +17,7 @@
 DOCUMENTATION = """
 ---
 module: os10_command
-version_added: "2.2"
+version_added: "2.9.6"
```

Comment:
You can remove all the `version_added:` strings for all modules and all module's options.